core/includes/install.core.inc

```diff
@@ -165,6 +165,9 @@ function install_state_defaults() {
     'profile_info' => array(),
     // An array of available installation profiles.
     'profiles' => array(),
+    // An array of base profiles of the chosen installation profile
+    // (e.g. chosen profile > base_profiles[0] > base_profiles[1] > etc.)
```

Comment:
```suggestion
    // An array of base profiles of the chosen installation profile (e.g.:
    // chosen profile > base_profiles[0] > base_profiles[1] > etc.).
```

Reply:
I changed the ':' to a ',' to match other instances of 'e.g.' in this file.